apps/docs/pages/availability.mdx

```diff
@@ -8,8 +8,7 @@ title: Availability
 
 1. Go to [Your Availability](https://app.cal.com/availability) page within your cal.com account.
```

Comment:
@zomars I don't think we even use apps/docs now. Maybe we can delete the docs folder?

Reply:
Cool